speechmatics/cli_parser.py

```diff
@@ -238,6 +238,13 @@ def get_arg_parser():
         default=None,
         help=("Comma-separated list of expected languages for language identification"),
     )
+    config_parser.add_argument(
+        "--sentiment-analysis",
+        dest="sentiment_analysis",
+        action="store_true",
+        default=False,
+        help="Whether to perform sentiment analysis on the transcript",
```

Comment:
nit: This can just be "Perform sentiment analysis on the transcript" since you can't use the flag to turn it off

Reply:
done